examples/tensorflow/question-answering/utils_qa.py

```diff
@@ -0,0 +1,425 @@
+# coding=utf-8
```

Comment:
Is this a direct copy of the PyTorch utils_qa.py? If yes, we should find a way to have both stay consistent with each other.

Reply:
It is! I know we talked about this before, but what's the right way to do that? I don't think we can symlink the same file in two places with git repos, but do we have some kind of macros we use to keep them consistent?

Reply:
We have utilities that check copies internally, will add this to it when I have a bit of time.

Reply:
Cool, thanks! I'll try to learn the syntax from how you do it and do it myself in future.